### PR TITLE
[SMTP Integration tests] RabbitMQ + Swift + Cassandra

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by chibenwa** [comment]
Yes I have a gitignore on my local machine on it.

Please commit it for me ...

**Comment by rouazana** [comment]
> Please commit it for me ...

done

**Comment by chibenwa** [comment]
```
[2244bf4b2c036e925885180640baae97c9f6b431] [ERROR]   MailQueueContract.concurrentEnqueueDequeueShouldNotFail:387 » NotTerminated
```

**Comment by chibenwa** [comment]
```
[2244bf4b2c036e925885180640baae97c9f6b431] [ERROR]  MailQueueContract.concurrentEnqueueDequeueShouldNotFail:387 » NotTerminated
```

test this please

**Comment by trantienduchn** [comment]
sorry I changed PR names to make people avoid confusing between this and #1911

**Comment by trantienduchn** [comment]
```
[0e6c297f4506d4881b8089f2fb857007b1512073] [ERROR] Errors: 
[0e6c297f4506d4881b8089f2fb857007b1512073] [ERROR]   JmapJamesServerContract.connectJMAPServerShouldRespondBadRequest » Runtime jav...
[0e6c297f4506d4881b8089f2fb857007b1512073] [ERROR]   JamesServerContract.connectOnSecondaryIMAPServerIMAPServerShouldSendShabangOnConnect » Runtime
[0e6c297f4506d4881b8089f2fb857007b1512073] [ERROR]   JamesServerContract.connectPOP3ServerShouldSendShabangOnConnect » Runtime java...
[0e6c297f4506d4881b8089f2fb857007b1512073] [ERROR]   JamesServerContract.hostnameShouldBeRetrievedWhenRestarting:59 » Provision Una...
[0e6c297f4506d4881b8089f2fb857007b1512073] [INFO] 
[0e6c297f4506d4881b8089f2fb857007b1512073] [ERROR] Tests run: 35, Failures: 0, Errors: 4, Skipped: 0
```

**Comment by chibenwa** [comment]
test this please

**Comment by trantienduchn** [comment]
```
[0e6c297f4506d4881b8089f2fb857007b1512073] [ERROR] Errors: 
[0e6c297f4506d4881b8089f2fb857007b1512073] [ERROR]   JmapJamesServerContract.connectJMAPServerShouldRespondBadRequest » Runtime jav...
[0e6c297f4506d4881b8089f2fb857007b1512073] [ERROR]   JamesServerContract.connectOnSecondaryIMAPServerIMAPServerShouldSendShabangOnConnect » Runtime
[0e6c297f4506d4881b8089f2fb857007b1512073] [ERROR]   JamesServerContract.connectPOP3ServerShouldSendShabangOnConnect » Runtime java...
[0e6c297f4506d4881b8089f2fb857007b1512073] [ERROR]   JamesServerContract.hostnameShouldBeRetrievedWhenRestarting:59 » Provision Una...
```

**Comment by trantienduchn** [comment]
I am fixing it. Related to swift clean up performer

**Comment by trantienduchn** [comment]
@chibenwa please take this https://github.com/linagora/james-project/pull/1911/commits/2c2364fb21509ee8d977ffede2eff73953182a87

in some tests, james server start, stop serveral times causing exception because Swift `BlobStore` doesnt allow to create container with same name many times. Update Swift CleanUp Performer from clean container content to delete container.

**Comment by trantienduchn** [comment]
```
[de6b38104cfd7fc01058edd695e49cdbf6a435e0] [ERROR] Errors: 
[de6b38104cfd7fc01058edd695e49cdbf6a435e0] [ERROR]   MailQueueContract.concurrentEnqueueDequeueShouldNotFail:387 » NotTerminated
[de6b38104cfd7fc01058edd695e49cdbf6a435e0] [INFO] 
[de6b38104cfd7fc01058edd695e49cdbf6a435e0] [ERROR] Tests run: 183, Failures: 0, Errors: 1, Skipped: 0
```
test this please

**Comment by trantienduchn** [comment]
```
[de6b38104cfd7fc01058edd695e49cdbf6a435e0] [ERROR] Errors: 
[de6b38104cfd7fc01058edd695e49cdbf6a435e0] [ERROR]   ForwardSmtpTest.forwardingAnEmailShouldWork » Runtime java.io.FileNotFoundExce...
[de6b38104cfd7fc01058edd695e49cdbf6a435e0] [ERROR]   SmtpStarttlsCommandTest.shouldNotRejectContentStartsWithStartTls » Runtime jav...
[de6b38104cfd7fc01058edd695e49cdbf6a435e0] [ERROR]   SmtpStarttlsCommandTest.shouldNotRejectContentWithStartTls » Runtime java.io.F...
[de6b38104cfd7fc01058edd695e49cdbf6a435e0] [ERROR]   SmtpStarttlsCommandTest.shouldNotRejectRcptWithStartTls » Runtime java.io.File...
[de6b38104cfd7fc01058edd695e49cdbf6a435e0] [ERROR]   SmtpStarttlsCommandTest.starttlsShouldBeRejectedWhenFollowedByCommand » Runtime
[de6b38104cfd7fc01058edd695e49cdbf6a435e0] [ERROR]   SmtpStarttlsCommandTest.starttlsShouldWork » Runtime java.io.FileNotFoundExcep...
[de6b38104cfd7fc01058edd695e49cdbf6a435e0] [INFO] 
[de6b38104cfd7fc01058edd695e49cdbf6a435e0] [ERROR] Tests run: 6, Failures: 0, Errors: 6, Skipped: 0
```
logs from last error

**Comment by chibenwa** [comment]
I know I dropped a commit adding the keystore by mistake.

**Comment by trantienduchn** [comment]
```
[d89ec8e751a90ea35362edd01bf81177ae30acc7] [ERROR] Failures: 
[d89ec8e751a90ea35362edd01bf81177ae30acc7] [ERROR]   ManageableMailQueueContract.getSizeShouldReturnMessageCountWhenSeveralMails:71 expected:<[2]L> but was:<[1]L>
[d89ec8e751a90ea35362edd01bf81177ae30acc7] [INFO] 
[d89ec8e751a90ea35362edd01bf81177ae30acc7] [ERROR] Tests run: 183, Failures: 1, Errors: 0, Skipped: 0
```
test this please

**Comment by chibenwa** [comment]
test this please

Same than previous

**Comment by chibenwa** [comment]
test this please

File: mpt/impl/smtp/cassandra-rabbitmq-swift/pom.xml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    <parent>
+        <groupId>org.apache.james</groupId>
+        <artifactId>apache-james-mpt-smtp</artifactId>
+        <version>3.3.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>apache-james-mpt-smtp-cassandra-rabbitmq-swift</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Apache James MPT SMTP Cassandra</name>
```
Comment (trantienduchn) [inline mpt/impl/smtp/cassandra-rabbitmq-swift/pom.xml]: It's missing rabbit + swift

**Comment by chibenwa** [comment]
`[21d73b030f21332270fcb84e9284f9ddb2fac263] [ERROR]   VacationIntegrationTest.jmapVacationShouldNotSendNotificationTwice:240 » ConditionTimeout`

(memory)
test this please

**Comment by chibenwa** [comment]
`[21d73b030f21332270fcb84e9284f9ddb2fac263] [ERROR]   RabbitMQMailQueueConfigurationChangeTest.increasingBucketCountShouldAllowBrowsingAllQueueElements:156 `

test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**